evap/staff/tests/test_tools.py

```diff
@@ -217,6 +221,73 @@ def test_do_nothing_if_test_run(self):
         self.assertEqual(len(messages), 4)
 
 
+class RemoveParticipationDueToInactivityTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.user = baker.make(UserProfile)
+        six_months_ago = datetime.today() - timedelta(days=6 * 30)
+        cls.evaluation = baker.make(
+            Evaluation,
+            state=Evaluation.State.PUBLISHED,
+            vote_start_datetime=six_months_ago - settings.PARTICIPATION_DELETION_AFTER_INACTIVE_TIME,
+            vote_end_date=six_months_ago.date(),
+            participants=[cls.user],
+        )
+        cls.evaluation.course.semester.archive()
+
+    @override_settings(PARTICIPATION_DELETION_AFTER_INACTIVE_TIME=timedelta(6 * 30))
+    def test_remove_user_due_to_inactivity(self):
+        self.assertTrue(self.user.evaluations_participating_in.exists())
+
+        messages = remove_inactive_participations(self.user)
+
+        self.assertFalse(self.user.evaluations_participating_in.exists())
+        self.assertTrue(self.user.can_be_marked_inactive_by_manager)
+        self.assertEqual(messages, [f"Removed {self.user.full_name} from 1 participation(s) due to inactivity."])
+
+        messages = remove_inactive_participations(self.user)
+
+        self.assertEqual(messages, [])
+
+    @patch("evap.evaluation.models.UserProfile.is_active", True)
+    @patch("evap.evaluation.models.UserProfile.can_be_marked_inactive_by_manager", True)
+    def test_do_not_remove_user_due_to_inactivity_with_recently_archived_evaluation(self):
```

Comment:
This relies on the default setting value, but it would be nice if the test passed even if some developers locally changed the setting. Thus, we should also

```suggestion
    @override_settings(PARTICIPATION_DELETION_AFTER_INACTIVE_TIME=timedelta(360))
    def test_do_not_remove_user_due_to_inactivity_with_recently_archived_evaluation(self):
```